MI-206 Added the distance travelled metrics to the db

AlineTheAlien opened the pull request:
Added the distance traveled metrics to the Ride table in the database. For each ride, there will be a distance traveled value that will be stored when the ride is over.
Also added method to set the distance traveled metric, to facilitate use from the application-side. It will allow the developers to quickly set the distance traveled value at the end of a ride.

AlineTheAlien committed:
MI-206 Added the distance travelled metrics to the db

Added the distance travelled metrics to the Ride table in the database. For each ride, there will be a distance travelled value thatwill be stored when the ride is over.
Also added method to set the distance travelled metric, to facilitate use from the application-side. It will allow the developers to quickly set the distance travelled value at the end of a ride.

ghost commented:
The implementation, structure and variable names are good. Don't forget your XML comments

AlineTheAlien committed:
MI-226 Added XML comments for RideCatalog and RideController classes

These two classes didn't have any XML tag comments for their methods, therefore I included the addition for this commit.

AlineTheAlien commented:
@NassimElSayed I have just included the XML comments, thank you!
